### ISSUE TITLE
NeedHelp in checkout function.

### ISSUE THREAD

**Issue by NiteeshK21** [NiteeshK21/HotelManagement#1]
I need a method to authorise access of checkout function only to staff members using password or something

**Comment by suman-somu** [comment]
The whole point of the program is to eliminate the staff processes, by a program. 
So, I think the program doesn't need to provide the check function only to staff members.

**Comment by NiteeshK21** [comment]
> The whole point of the program is to eliminate the staff processes, by a program. So, I think the program doesn't need to provide the check function only to staff members.

Yes, the program aims at reducing staff processes but there has to be a admin for verifying data and payments. Checkout function is to verify the bills of customer and checkout the customer if everything is clear.
